mirrord-layer/src/common.rs

```diff
@@ -29,3 +38,172 @@ pub enum HookMessage {
     File(HookMessageFile),
     GetAddrInfoHook(GetAddrInfoHook),
 }
+
+mod common_ops {
```

Comment:
I suggest moving these modules under a separate thing, this `common` module name is already not very helpful. Maybe have a `common_detour` folder/module, and inside have the `hooks` and `ops`, so we access those as `common_detour::hooks` and `common_detour::ops`.